### PR TITLE
negative values are inefficient and not recommended

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Enumerator constants must be in the range of a 32-bit integer. Since enum values use varint encoding on the wire, negative values are inefficient and thus not recommended.

### CODE REVIEW

**Comment by disksing** [comment]
LGTM

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -25,9 +25,9 @@ message KeyError {
 }
 
 enum CommandPri {
-    Normal = 0;
-    Low = -1;
-    High = 1;
+    Normal = 0; // Normal must the default value
```
Comment (siddontang) [inline proto/kvrpcpb.proto]: strange to see low has a bigger value than normal...

Comment (zhangjinpeng1987) [inline proto/kvrpcpb.proto]: It just a number and we will not use the number but the name. The important point is that negative is inefficient.

Comment (siddontang) [inline proto/kvrpcpb.proto]: do we have many cases to use low priority? if not, the negative is not a problem.

Comment (zhangjinpeng1987) [inline proto/kvrpcpb.proto]: Big query like AP will use low priority.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**